tensordict/__init__.py

```diff
@@ -9,6 +9,8 @@
     merge_tensordicts,
     SubTensorDict,
     TensorDict,
+    detect_loop,
+    _TensorDictKeysView,
```

Comment:
I think we don't want to expose this, if someone really wants to use it then they can import from `tensordict.tensordict`.

Reply:
Alright but we need to import it to the test file in order to run the tests